pom.xml

```diff
@@ -23,6 +23,11 @@
             <artifactId>commons-configuration</artifactId>
         </dependency>
         <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-csv</artifactId>
+            <version>1.1</version>
```

Comment:
Alwasy check the parent poms to see if a version was already configured for your dependency. In this case it was.